### DISCUSSION TITLE
Comunidade do Painel e-SUS APS!

### DISCUSSION THREAD

**Discussion by viniciusfiocruz** [CampusVirtualFiocruz/painel-esus#1] [Announcements]
## 👋 Seja bem-vindo(a)!

Esse espaço de discussões será utilizado para desenvolvermos painéis de análise de dados do e-SUS APS colaborativamente.

  * Sugira temas para relatórios e painéis.
  * Dê idéias para melhorar a navegação.
  * Sugira indicadores e recomendações de uso das informações já existentes.
  * Seja gentil e aberto, para construirmos uma comunidade forte para o SUS 💪.

Comece se apresentando na sua primeira postagem!

**Comment by nicholassampaio22** [comment]
Boa tarde, Sou Nicholas Analista de sistema e estou muito feliz com o projeto do painel, porém analisando o sistema percebi uns erros ao tentar acessar a aba idosos ela não abri e olhando os logs apresenta esse erros em anex.

Estou rodando a versão 0.9.8 em docker.

[erros painel gerar modulo idoso.txt](https://github.com/user-attachments/files/21477356/erros.painel.gerar.modulo.idoso.txt)

**Reply by talesmota** [reply]
Bom dia @nicholassampaio22, seja bem vindo!

Abre uma [issue](https://github.com/CampusVirtualFiocruz/painel-esus/issues/new/choose) por favor, pra que fique documentado para a comunidade essa nossa atuação, pode ser?
Eu olhei o log aqui e percebi que há uma categoria 'não informada' sendo retornada pelo PEC. Essa categoria não está mapeada no dicionário interno do relatório do idoso. Eu vou fazer uns teste aqui e te dou um retorno pra podermos testar a solução. Aí continuamos pela issue, ok?

At.te

**Comment by amnesiaoliveira** [comment]
Boa tarde, Meu nome é Eugenio, antes nós usáva-mos o painel no nosso município e agora estamos querendo usar novamente. Alguém poderia nos auxiliar para colocar o sistema para rodar novamente, pois é uma aplicação muito boa para realizarmos alguns gerenciamentos.
